my_file.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+if [ -n "$1" ]; then
```

Comment:
Instead of checking if an argument is provided using if [ -z "$1" ], you can use the ${1:-50} syntax directly in the assignment. This will set the default value to 50 if no argument is provided. For example: num_files=${1:-50}.